assets/src/scss/new-identity/style.scss

```diff
@@ -15,6 +15,9 @@
   --button-primary--color: var(--grey-900);
   --button-primary--visited--color: var(--grey-900);
   --button-primary--hover--color: var(--grey-900);
+  --comment-reply-link--color: var(--grey-900);
```

Comment:
I would remove all those link-related variables here, since they should be fixed by https://github.com/greenpeace/planet4-master-theme/pull/1966